src/settings/mod.rs

```diff
@@ -41,6 +65,8 @@ pub struct Smtp
 #[derive(Debug, Deserialize)]
 pub struct Settings
 {
+  pub authdb: AuthDb,
+  pub bouncelimits: BounceLimits,
```

Comment:
Is the alllowercase capitalization on these a by-product of the code that automagically populates them from environment variables, or just a stylistic choice?

Reply:
The former. If we insert underscores in the keys here, we can't set them as environment variables unless we use some other separator at that level. (and actually, I'm not sure what the valid separators would even be there)